i have no idea what upstream meant by [adding those][1],
and i'm not sure if they did either;

anyway, upstream is gone for over 6 years,
build now fails with python 3.12 headers, so let's remove it altogether

[1]: https://bazaar.launchpad.net/~onboard/onboard/trunk/revision/1883

diff --git a/setup.py b/setup.py
index 27000f3..76abe04 100755
--- a/setup.py
+++ b/setup.py
@@ -230,5 +230,3 @@ class Extension_osk(Extension):
                            extra_compile_args = [
-                               "-Wsign-compare",
-                               "-Wdeclaration-after-statement",
-                               "-Werror=declaration-after-statement"],
+                               "-Wsign-compare"],
 
